source/NVDAObjects/IAccessible/ia2Web.py

```diff
@@ -68,6 +68,16 @@ def _get_states(self):
 			states.discard(controlTypes.STATE_EDITABLE)
 		return states
 
+	def getFocusInfo(self):

```

Comment:
May be this can go on the IAccessible base class?

Reply:
I'm hesitant to move this down if it's not used. Initially I'd prefer to focus on solving the specific problem rather than trying to generalize it.